### ISSUE TITLE
Support AWS SSO login

### ISSUE THREAD

**Issue by lloydpick** [mercury2269/sqsmover#25]
We use [AWS SSO](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sso.html) to login to our accounts, however it seems that sqsmover doesn't understand how to login to those accounts.

```   
⨯ Failed to resolve source queue. Error: no valid providers in chain. Deprecated.
For verbose messaging see aws.Config.CredentialsChainVerboseErrors
```

The work around for us at the moment is to use [linaro-its/aws2-wrap](https://github.com/linaro-its/aws2-wrap) to essentially feed the creds into sqsmover, but it would be great if it could understand how to use SSO credentials.

**Comment by szymon3** [comment]
@lloydpick when performing SSO login, do you specify AWS Profile? If so, are you providing `--profile` with that profile name to sqsmvoer?

**Comment by lloydpick** [comment]
Yeah

<img width="1664" alt="Screen_Shot_2021-05-12_at_9_08_50_AM" src="https://user-images.githubusercontent.com/71318/117980365-e566f080-b301-11eb-9f02-196c844e6024.png">

**Comment by mercury2269** [comment]
Should be resolved with 0.4.0 release.
